mlir/lib/Bindings/Python/RegisterEverything.cpp

```diff
@@ -7,20 +7,17 @@
 //===----------------------------------------------------------------------===//
 
 #include "mlir-c/RegisterEverything.h"
-#include "mlir-c/Conversion.h"
-#include "mlir-c/Transforms.h"
-
 #include "mlir/Bindings/Python/PybindAdaptors.h"
 
 PYBIND11_MODULE(_mlirRegisterEverything, m) {
-  m.doc() = "MLIR All Upstream Dialects and Passes Registration";
+  m.doc() = "MLIR All Upstream Dialects, Translations and Passes Registration";
 
   m.def("register_dialects", [](MlirDialectRegistry registry) {
     mlirRegisterAllDialects(registry);
   });
+  m.def("register_translations",
```

Comment:
Can we name this `register_llvm_translations`?

Reply:
You're right, I was confused with `registerAllTranslation`

Reply:
Maybe we should name it `register_all_translations` and actually register them all instead of just export to LLVM?

Reply:
I'm fine with making that change, the only reason I've only made llvm translation was, `mlirRegisterAllLLVMTranslations` is currently the only available method in C-API to register the translations.
If nothing blocks, I'd add more C-API functions for the translations and then do register_all_translation in python.

Reply:
Currently there's no 'pass' for the translations other than `to llvm`. Only `convert-gpu-launch-to-vulkan-launch` calls spirv::serialize at the moment but it's only a part of vulkan conversion.
I'd say, llvm exporting is all needed for now.

I'm going to discuss creating a new pass for spirv serialization in line with the recent change in gpu codegen pipeline.
Please merge this @joker-eph if that makes sense to you, I don't have a permission.